app/Console/Commands/BulkReviewPosts.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+
+class BulkReviewPosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:bulkreviewposts {campaign} {oldStatus} {newStatus} {--type= : Filter by the type of post} {--logfreq=1000} {--log} {--tag=* : Tag(s) slug to tag post by. e.g. hide-in-gallery}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Bulk review and tag posts constrained by the provided parameters';
+
+    /**
+     * The Post Manager instance.
+     *
+     * @var Rogue\Managers\PostManager
+     */
+    protected $posts;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostManager $posts)
+    {
+        parent::__construct();
+
+        $this->posts = $posts;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:bulkreviewposts: Starting to bulk accept posts!');
+
+        $postType = $this->option('type') ?? null;
+        $logfreq = $this->option('logfreq');
+        $log = $this->option('log');
+        $tags = $this->option('tag') ?? null;
+
+        $query = (new Post)->newQuery();
+        $query = $query->where('campaign_id', $this->argument('campaign'));
+        $query = $query->where('status', $this->argument('oldStatus'));
+
+        if ($postType !== null) {
+            $query = $query->where('type', $postType);
+        }
+
+        $posts = $query->get();
+
+        if ($posts->isNotEmpty()) {
+            foreach ($posts as $post) {
+                if ($post->id % $logfreq == 0) {
+                    info('rogue:bulkreviewposts: accepting and tagging post: ' . $post->id);
```

Comment:
shouldn't this log message be hidden under the `$log` flag?

Reply:
@sbsmith86 great q! This is specific to logging in the command and `info('rogue:bulkreviewposts: accepting and tagging post: ' . $post->id);` which is controlled by `$logfreq`. 

The `$log` flag sends to the `PostManager` and hides logging that is done there.

Reply:
ok! can you clarify that in a comment? Also maybe in the command signature. It's a little confusing to read.

Reply:
yes good call! Updated :)